packages/thirdweb/src/wallets/in-app/core/lib/in-app-wallet-calls.ts

```diff
@@ -0,0 +1,99 @@
+import type { ThirdwebClient } from "../../../../client/client.js";
+import { eth_getTransactionReceipt } from "../../../../rpc/actions/eth_getTransactionReceipt.js";
+import { getRpcClient } from "../../../../rpc/rpc.js";
+import { sendAndConfirmTransaction } from "../../../../transaction/actions/send-and-confirm-transaction.js";
+import { sendBatchTransaction } from "../../../../transaction/actions/send-batch-transaction.js";
+import { type Hex, generateRandomHex } from "../../../../utils/encoding/hex.js";
+import type { PreparedSendCall } from "../../../eip5792/send-calls.js";
+import type {
+  GetCallsStatusResponse,
+  WalletCallReceipt,
+  WalletSendCallsId,
+} from "../../../eip5792/types.js";
+import type { Account, Wallet } from "../../../interfaces/wallet.js";
+
+const bundlesToTransactions = new Map<string, Hex[]>();
```

Comment:
this map could get huge in a long running app. would use a bounded map here or a LRU